packages/sdk/Dockerfile

```diff
@@ -169,5 +169,25 @@ ADD --chmod=644 \
 RUN tar -xJf /usr/share/cartesi-machine/images/linux-headers.tar.xz -C / && \
     rm /usr/share/cartesi-machine/images/linux-headers.tar.xz
 
+# Install migrate db cli
+ARG MIGRATE_VERSION=v4.18.2 
```

Comment:
This should be defined at `packages/sdk/docker-bake.hcl`, and use the already existent `CARTESI_ESPRESSO_READER_VERSION`.

You'll need to change the target branch to `prerelease/v2-alpha`.

Reply:
Sorry for the confusion, this is not from cartesi-espresso-reader.

Reply:
So I suggest naming it `GO_MIGRATE_VERSION` and expect its value to not have the initial `v`.

Reply:
I added GO_MIGRATE_VERSION to the docker-bake.hcl file and also sorted the list.